### PR TITLE
chore: update support matrix

### PR DESCRIPTION
**Pull request by sumo-drosiek**
chore: add support for KOPS 1.26, EKS 1.25; remove support for GKE 1.21, EKS 1.21

Start supporting 1.26

### Checklist

- [ ] Changelog updated or skip changelog label added
- [ ] Documentation updated
- [ ] Template tests added for new features
- [ ] Integration tests added or modified for major features

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -86,9 +86,9 @@ The following table displays the tested Kubernetes and Helm versions.
 
 | Name          | Version                                  |
 | ------------- | ---------------------------------------- |
-| K8s with EKS  | 1.21<br/>1.22<br/>1.23<br/>1.24          |
-| K8s with Kops | 1.22<br/>1.23<br/>1.24<br/>1.25          |
-| K8s with GKE  | 1.21<br/>1.22<br/>1.23<br/>1.24<br/>1.25 |
+| K8s with EKS  | 1.22<br/>1.23<br/>1.24<br/>1.25          |
+| K8s with Kops | 1.22<br/>1.23<br/>1.24<br/>1.25<br/>1.26 |
+| K8s with GKE  | 1.22<br/>1.23<br/>1.24<br/>1.25          |
 | K8s with AKS  | 1.23<br/>1.24<br/>1.25                   |
```
Comment (sumo-drosiek) [inline docs/README.md:92]:
```suggestion
| K8s with AKS  | 1.24<br/>1.25<br/>1.26                   |
```

File: .changelog/2969.changed.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+chore: add support for KOPS 1.26, EKS 1.25; remove support for GKE 1.21, EKS 1.21
```
Comment (sumo-drosiek) [inline .changelog/2969.changed.txt:1]:
```suggestion
chore: add support for KOPS 1.26, EKS 1.25, AKS 1.26; remove support for GKE 1.21, EKS 1.21, AKS 1.23
```

File: .changelog/2969.changed.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+chore: add support for KOPS 1.26, EKS 1.25, AKS 1.26; remove support for GKE 1.21, EKS 1.21, AKS 1.23
```
Comment (sumo-drosiek) [inline .changelog/2969.changed.txt:1]:
```suggestion
chore: add support for KOPS 1.26, EKS 1.25, AKS 1.26, OpenShift 4.12; remove support for GKE 1.21, EKS 1.21, AKS 1.23
```

**Comment by aboguszewski-sumo** [comment]
Since we're adding support for `1.26` and dropping for `1.21`, please update `tests/integration/kind_images.json`

**Review by aboguszewski-sumo — APPROVED** [review]

**Pull request merged**